src/languages/autoit.js

```diff
@@ -1,25 +1,46 @@
 /*
 Language: AutoIt
 Author: Manh Tuan <junookyo@gmail.com>
+Modified: Michał Lipok
 Description: AutoIt language definition
 Category: scripting
 */
 
 /** @type LanguageFn */
 export default function(hljs) {
   const KEYWORDS = 'ByRef Case Const ContinueCase ContinueLoop ' +
-        'Default Dim Do Else ElseIf EndFunc EndIf EndSelect ' +
+        'Dim Do Else ElseIf EndFunc EndIf EndSelect ' +
         'EndSwitch EndWith Enum Exit ExitLoop For Func ' +
         'Global If In Local Next ReDim Return Select Static ' +
         'Step Switch Then To Until Volatile WEnd While With';
 
-  const LITERAL = 'True False And Null Not Or';
+  const DIRECTIVES = [
+    "EndRegion",
+    "forcedef",
+    "forceref",
+    "ignorefunc",
+    "include",
+    "include-once",
+    "NoTrayIcon",
+    "OnAutoItStartRegister",
+    "pragma",
+    "Region",
+    "RequireAdmin",
+    "#Tidy_Off",
+    "#Tidy_On",
+    "#Tidy_Parameters"
```

Comment:
These will never work unless you add `$pattern` to your `keywords` hash.  The default keyword matcher is `\w+` which will NEVER match the #.  I think `/#?\w+/` might work.  See `powershell.js` for an example of doing this.

Reply:
```suggestion
    "Tidy_Off",
    "Tidy_On",
    "Tidy_Parameters"
```
?

Reply:
wWell why did you have the # in front of them in the first case? How are they used in code? Is the  doubled?   I assumed you'd see:

```
##Tidy_Off
```

Otherwise I wasn't sure why you added the extra `#` at all...

And you'd still need $pattern to deal with the `-`...

Reply:
by mistake.